apps/api/internal/auth/password_test.go

```
@@ -0,0 +1,122 @@
+		PublicOrigin: "https://contentflow.example", OwnerIssuer: "issuer", OwnerSubject: "owner",
+		WorkspaceID: "workspace", CredentialKey: make([]byte, 32),
+	}, &fakeProvider{}, store)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return service, store
+}
+
+func signIn(service *Service, email, password string) *httptest.ResponseRecorder {
+	body, _ := json.Marshal(map[string]string{"email": email, "password": password})
+	request := httptest.NewRequest(http.MethodPost, "/api/v1/auth/password", strings.NewReader(string(body)))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)